lib/std/os.zig

```diff
@@ -44,6 +44,32 @@ pub const windows = @import("os/windows.zig");
 pub const posix_spawn = @import("os/posix_spawn.zig");
 pub const ptrace = @import("os/ptrace.zig");
 
+/// Pipe read side
+pub const pipe_rd = 0;
+/// Pipe write side
+pub const pipe_wr = 1;
+
+pub const windowsPtrDigits: usize = std.math.log10(math.maxInt(usize));
+pub const otherPtrDigits: usize = std.math.log10(math.maxInt(u32)) + 1; // +1 for sign
+pub const handleCharSize = if (builtin.target.os.tag == .windows) windowsPtrDigits else otherPtrDigits;
+
+pub fn handleToString(handle: fd_t, buf: []u8) std.fmt.BufPrintError![]u8 {
```

Comment:
name suggestions other than handleToString? Its called handle on Windows and file descriptor on Unixoids.
For now I used the shorter name.